src/migrations/20220409183111_deprecate_old_shuffle.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+exports.up = function (knex) {
+    return knex.raw(
```

Comment:
Alternatively, you can delete all shuffletype `game_options`

Reply:
Would rather not delete since the existence of the option tells us a little bit about how many servers have played since the option was introduced